openbb_platform/CONTRIBUTING.md

```diff
@@ -483,14 +483,14 @@ You can expect the following endpoint structure when using a `Fetcher` to serve
 
 ```python
 @router.command(model="Example")
-def model_example(
+async def model_example(    # create an async endpoint
     cc: CommandContext,
     provider_choices: ProviderChoices,
     standard_params: StandardParams,
     extra_params: ExtraParams,
 ) -> OBBject[BaseModel]:
     """Example Data."""
-    return OBBject(results=Query(**locals()).execute())
+    return await OBBject.from_query(Query(**locals()))
```

Comment:
Out of curiosity, is there a reason for us doing the query execution inside the OBBject (with the `from_query` method)? 
Not sure if I like it, but at the same time feels easier to read ("creates a OBBject from a query"). :eyes:

Reply:
it was mostly to look a little cleaner and easier to follow since otherwise would probably have had to do

```python
return OBBject(results=await Query(**locals()).execute())
````

🤣

Reply:
I prefer the `from_query` because its not confusing then

Reply:
gotcha! @piiq also mentioned that `from_something` aligns with pydantic's syntax.